LICENSE

```diff
@@ -0,0 +1,25 @@
+simp_consul - 
+
+Per Section 105 of the Copyright Act of 1976, these works are not entitled to
```

Comment:
This section can go. This is all non-government and copyright held by Onyx Point, Inc.

Reply:
Ok